options/style.css

```diff
@@ -1,7 +1,26 @@
 .form {
+  font-family: 'Gill Sans';
```

Comment:
As the add-on preferences page is displayed in the Firefox interface, I think we should keep default font.